server/plugins/connect-plugin-ob-mysql/src/main/java/com/oceanbase/odc/plugin/connect/obmysql/OBMySQLConnectionExtension.java

```diff
@@ -142,10 +142,10 @@ protected TestResult internalTest(String jdbcUrl, Properties properties, int que
         } catch (Exception e) {
             Throwable rootCause = ExceptionUtils.getRootCause(e);
             if (rootCause == null) {
-                log.warn("Failed to get connection, errMsg={}", e.getLocalizedMessage());
+                log.warn("Failed to get connection, errMsg={}", e.getLocalizedMessage(), e);
```

Comment:
there may be a lot of log printing

Reply:
may refer com.oceanbase.odc.common.util.ExceptionUtils.getRootCauseReason, which record 5 depth stack as root cause